packages/editor/src/components/post-preview-button/test/__snapshots__/index.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`PostPreviewButton render() should render currentPostLink otherwise 1`] = `
 <ForwardRef(Button)
   className="editor-post-preview"
-  disabled={false}
+  disabled={true}
```

Comment:
The tests still reference the now-defunct `isSaveable` and should be updated.

Reply:
👍I've ended up switching the PostPreviewButton prop to `isDisabled` which I think is tidier. Also updated the tests this time.

Reply:
I've now reintroduced `isSaveable` and sorted the tests.